src/main/java/br/codenation/hospital/manager/exception/HospitalException.java

```diff
@@ -0,0 +1,7 @@
+package br.codenation.hospital.manager.exception;
+
+public class HospitalException extends RuntimeException {
+
+    public HospitalException(String message){ super(message); }
```

Comment:
Estava lendo [aqui](https://www.baeldung.com/java-chained-exceptions) e agora entendi o motivo - pra que as exceptions sejam encadeadas, é preciso um construtor com o Throwable também

```suggestion
    public HospitalException(String message){ super(message); }
    
    public HospitalException(String message, Throwable throwable) { super(message, throwable); }
```